sherpa/utils/__init__.py

```diff
@@ -2641,6 +2642,13 @@ def is_iterable(arg):
     return isinstance(arg, (list, tuple, np.ndarray)) or np.iterable(arg)
 
 
+# Can this return TypeGuard[Sequence]?
```

Comment:
This is a one line function that is used once. Can we avoid this level of indiraction and just put it (wiht the comment) directly in line where it is used?
We can refractor is needed later.

Reply:
I meant this comment to apply to the entire function, not this one comment line.

Reply:
Well, I plan to use this in the follow on commit #1712 but I'm not sure whether I'll be able to get it done in time is to be seen.